crates/kitsune_p2p/dht/src/region_set/ltcs.rs

```diff
@@ -15,8 +14,9 @@ use super::{Region, RegionCoords, RegionData, RegionDataConstraints};
 /// [`SpaceSegment`]s are implied by the [`ArqBoundsSet`].
 ///
 /// LTCS stands for Logarithmic Time, Constant Space.
-#[derive(Debug, PartialEq, Eq, derive_more::Constructor, serde::Serialize, serde::Deserialize)]
-#[cfg_attr(feature = "test_utils", derive(Clone))]
+#[derive(
+    Debug, Clone, PartialEq, Eq, derive_more::Constructor, serde::Serialize, serde::Deserialize,
```

Comment:
before this, `Clone` was only available used for tests. what's the reason to lift that?

Reply:
I got as far as this struct and got stuck with this kind of question when I tried to remove test_utils by default.

If I remember rightly the production code is relying on Clone for this struct. Whether that's valid is hard to say. It looks immutable so it's probably safe to copy its data but whether it should be copied I don't know.

Reply:
I'm not sure why I couldn't get this working originally either, but thanks for pointing this out, because after merging in another branch which fiddles with features (#2849), I was able to reinstate the cfg_attr :+1:

Reply:
Oops, spoke too soon, Clone is necessary in prod (unless we were to do some extensive type changes around deserialization of network messages). It's not so bad though, this struct doesn't get larger than about 1KB.